### PR TITLE
feat: Added production automation for RAK devices

### PR DESCRIPTION
**Pull request by MuratUrsavas**
**Issue**
(NebraLtd/helium-miner-software#367)

- Summary:
The RAK devices with Nebra firmware are almost ready for prime time and we need to make sure our automated onboarding process includes them.

**How**
* Created a variant for certain GitHub actions.
* Since basically the RAK's are RPi's, we've merged the docker-compose file creation.
* The RAK devices images are pushed into their own fleets in Balena.

**Checklist**

- [ ] Tests added
- [x] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [x] Thought about variable and method names

### CODE REVIEW

File: .github/workflows/push-to-prod-draft.yml [hunk 1]
```diff
@@ -29,8 +29,8 @@ jobs:
           python gen_docker_compose.py rockpi -o docker-compose.yml
           ROCKPI="-rockpi"
           echo "ROCKPI=$ROCKPI" >> $GITHUB_ENV
-      - name: Generate Docker Compose for RaspberryPi
-        if: matrix.sbc == 'raspi'
+      - name: Generate Docker Compose for RaspberryPi & RAK
```
Comment (marvinmarnold) [inline .github/workflows/push-to-prod-draft.yml:32]: This should work, but not sure we want to make this the pattern for all vendors we support. They won't all map neatly to an existing Docker Compose template. Might be more maintainable to add a config for each vendor to gen_docker_compose, settings.ini, etc.

File: .github/workflows/push-to-prod-draft.yml [hunk 2]
```diff
@@ -45,6 +45,12 @@ jobs:
         with:
           balena_api_token: ${{secrets.BALENA_API_TOKEN_1}}
           balena_command: "deploy nebraltd/helium-${{ matrix.variant }}-${{ matrix.frequency }} --logs --debug --nocache --build --draft"
+      - name: Balena Deploy (RAK)
+        if: matrix.sbc == 'rak' && success()
+        uses: nebraltd/balena-cli-action@v13.1.10
+        with:
+          balena_api_token: ${{secrets.BALENA_API_TOKEN_2}}
+          balena_command: "deploy nebraltd/helium-rak-${{ matrix.frequency }} --logs --debug --nocache --build --draft"
```
[the remark this answers is not in the document]
Comment (MuratUrsavas) [reply .github/workflows/push-to-prod-draft.yml:53]: You're right. Will fix it.

File: .github/workflows/push-to-prod-draft.yml (hunk 1 above)
Comment (MuratUrsavas) [reply .github/workflows/push-to-prod-draft.yml:32]: Can we make it based on the case? In this one, it fits perfectly fine and adding more code could increase complexity. But if there will be any other one, that wouldn't fit perfectly, then we could create a new one for it. Deal? :smiley:

File: .github/workflows/push-to-prod-draft.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (MuratUrsavas) [reply .github/workflows/push-to-prod-draft.yml:56]: "RAK Outdoor" has been excluded from the matrix :+1:

**Review by cluutch — APPROVED** [review]

**Review by marvinmarnold — APPROVED** [review]

**Comment by marvinmarnold** [comment]
@MuratUrsavas can this be closed now? Looks like it worked: https://dashboard.balena-cloud.com/login

**Comment by MuratUrsavas** [comment]
Sure, I'd left it open for your attention per your request.

**Pull request merged**